### PR TITLE
Update BBEs for  ignoring return values and errors

### PR DESCRIPTION
**Pull request by prakanth97**

### CODE REVIEW

File: examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.bal [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import ballerina/io;
 
 public function main() {
-    // Allowed only if the return value is `()`.
+    // Allowed only if the return value is ().
```
Comment (MaryamZi) [inline examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.bal:4]: Why are we removing backticks?

File: examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
 
-Ballerina does not allow silently ignoring return values.
-To ignore a return value, assign it to `_`; this is like 
-an implicitly declared variable that cannot be referenced.
-When a return type includes an error, you have to do something 
-with the error.
-`_` is of type any: you cannot use `_` to ignore an error.
-`checkpanic` is like `check`, but panics on error rather than 
-returning.
+Ballerina does not allow silently ignoring return values. To ignore a return value, assign it to `_`. This is like an implicitly declared variable that cannot be referenced. `_` is any type that does not include `error`. You cannot use `_` to ignore an error. When a return type includes an error, you have to do something with the error.
```
Comment (MaryamZi) [inline examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md:3]: Why the change from ; ?

Comment (MaryamZi) [inline examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md:3]: `_` isn't a type, what we had originally (`_` is of type any) was correct.

File: examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md [hunk 3]
```diff
@@ -1,13 +1,8 @@
...
 
-Ballerina does not allow silently ignoring return values.
-To ignore a return value, assign it to `_`; this is like 
-an implicitly declared variable that cannot be referenced.
-When a return type includes an error, you have to do something 
-with the error.
-`_` is of type any: you cannot use `_` to ignore an error.
-`checkpanic` is like `check`, but panics on error rather than 
-returning.
+Ballerina does not allow silently ignoring return values. To ignore a return value, assign it to `_`; this is like an implicitly declared variable that cannot be referenced. When a return type includes an error, you have to do something with the error. `_` is of type any: you cannot use `_` to ignore an error.
```
Comment (praneesha) [inline examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md:3]:
```suggestion
Ballerina does not allow ignoring return values. To ignore a return value, assign it to `_`; this is like an implicitly declared variable that cannot be referenced. When a return type includes an error, you have to do something with the error. `_` is of type any: you cannot use `_` to ignore an error.
```

File: examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
+
+`checkpanic` is like `check`, but panics on error rather than returning.
```
Comment (praneesha) [inline examples/ignoring-return-values-and-errors/ignoring_return_values_and_errors.md:5]:
```suggestion
`checkpanic` is like `check`but it panics on the error rather than on returning.
```

**Review by praneesha — APPROVED** [review]
